src/components/pages/Profile.js

```diff
@@ -1,3 +1,26 @@
-const Profile = () => <div>Profile page under construction</div>;
+import { useSelector } from 'react-redux';
+
+import { selectAllJoinedRockets } from '../../redux/effects/rocketEffects';
+
+const Profile = () => {
+  const { rockets } = useSelector(selectAllJoinedRockets);
+  const isRocktesEmapty = rockets.length === 0;
+
+  return (
+    <section>
+      <div>
+        <h2>My Rockets</h2>
+        {!isRocktesEmapty && (
+          <ul>
+            {rockets.map(({ name, id }) => (
+              <li key={id}>{name}</li>
+            ))}
+          </ul>
+        )}
+        {isRocktesEmapty && <p>No Missions</p>}
```

Comment:
It's of no big impact, but, where you have `No  Missions`, it should be `No Rockets` since it's about rockets.